awesome_owl/static/src/todo_list/todo_list.js

```diff
@@ -0,0 +1,46 @@
+/** @odoo-module **/
+
+import { filterInPlace, useAutofocus } from "../utils";
+import { Component, useState } from "@odoo/owl";
+import { TodoItem } from "./todo_item";
+
+export class TodoList extends Component {
+    static template = "awesome_owl.TodoList";
+    static components = { TodoItem };
+
+    setup() {
+        this.todos = useState([
+            { id: 0, description: "commit murder", isCompleted: true },
+            { id: 1, description: "uwu", isCompleted: false },
+        ]);
+
+        useAutofocus("todo_input");
+    }
+
+    addTodo(ev) {
+        if (ev.keyCode !== 13) return;
+
+        const description = ev.target.value.trim();
+        let id = 0;
+        if (this.todos.length !== 0)
+            id = this.todos[this.todos.length - 1];
+
+        if (description === "") return;
```

Comment:
This condition should be done earlier, probably at the same time than the Keycode check
```suggestion
        if (!description) return;
```